sources/niLang/tsrc/Test_DataTable_Serialize.cpp

```diff
@@ -41,6 +41,92 @@ TEST_FIXTURE(FDTSerialize,Types) {
   CHECK_EQUAL(eDataTablePropertyType_Int32, dt->GetVarFromIndex(i).GetType());
 }
 
+TEST_FIXTURE(FDTSerialize,Path) {
+  Ptr<iDataTable> dtRoot = ni::CreateDataTable("Root");
+  Ptr<iDataTable> dtC1 = ni::CreateDataTable("Child1");
+  Ptr<iDataTable> dtC2 = ni::CreateDataTable("Child2");
+  Ptr<iDataTable> dtGC1 = ni::CreateDataTable("GrandChild1");
+  Ptr<iDataTable> dtGC2 = ni::CreateDataTable("GrandChild2");
+  Ptr<iDataTable> dtGC3 = ni::CreateDataTable("GrandChild3");
+  dtRoot->AddChild(dtC1);
+  dtRoot->AddChild(dtC2);
+  dtC1->AddChild(dtGC1);
+  dtC2->AddChild(dtGC2);
+  dtC2->AddChild(dtGC3);
+  // Root
+  // - Child1/@S="123"
+  // -- GrandChild1/@F=1.23f
+  // - Child2/@U=123
+  // -- GrandChild2/@B=eTrue
+  // -- GrandChild3/@V2=Vec2F(1,1,1)
+  // -- GrandChild3/@V3=Vec3F(1,1,1)
```

Comment:
You can also write this in XML and use `Ptr<iDataTable> ptrDT = CreateDataTableFromXML(_dtXML.c_str(),_dtXML.size());`.

It would probably be more readable. If there's a reason not to please add a comment.

Reply:
Done.